addOns/reports/src/main/zapHomeFiles/reports/traditional-html-plus/report.html

```diff
@@ -518,6 +560,160 @@ <h3 th:text="#{report.alerts.detail}">Alert Detail</h3>
 			<div class="spacer"></div>
 		</th:block>
 	</th:block>
+
+	<th:block th:if="${reportData.isIncludeSection('sequencedetails')}">
+		<th:block th:if="${reportData.isIncludeSection('sequencedetails')}">
```

Comment:
Seems duplicated.

Reply:
Good catch, either I missed a condition or it's a c&p error